modules/telco-core-reference-design-specification-for-product-title-product-version.adoc

```diff
@@ -0,0 +1,11 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/telco_core_ref_design_specs/telco-core-rds.adoc
+
+:_mod-docs-content-type: REFERENCE
```

Comment:
Same as one of the previous comments - consider making this an assembly. See [pr.netlify.app/openshift-enterprise/latest/scalability_and_performance/telco-core-rds](preview).